src/utils/stream.js

```diff
@@ -39,4 +41,15 @@ function createStreamPromise(stream, timeoutSeconds, bytesLimit = 1024 * 1024 *
   })
 }
 
-module.exports = { createStreamPromise }
+function bufferToStream(buffer) {
```

Comment:
Should we use the dependency directly?

https://github.com/sindresorhus/to-readable-stream/blob/master/index.js

This will ensure they update that code for us :)

Reply:
Good point. Actually found a couple of other unmaintained libs that does this, but didn't find this one.

Reply:
Fixed in 646288ffc18a924eba67028071071e302144799f